test/e2e_node/image_list.go

```diff
@@ -47,7 +47,8 @@ const (
 	maxParallelImagePullCount = 5
 
 	// SampleDevicePluginDSYAML is the path of the daemonset template of the sample device plugin. // TODO: Parametrize it by making it a feature in TestFramework.
-	SampleDevicePluginDSYAML = "test/e2e/testing-manifests/sample-device-plugin.yaml"
+	SampleDevicePluginDSYAML  = "test/e2e/testing-manifests/sample-device-plugin/sample-device-plugin-auto-registration.yaml"
+	SampleDevicePluginDS2YAML = "test/e2e/testing-manifests/sample-device-plugin/sample-device-plugin-control-registration.yaml"
```

Comment:
`DS2` is a bit cryptic :) Perhaps `SampleDevicePluginControlRegistrationDSYAML`?
In general let's pick a name which is a bit more explicit about the purpose, at cost of being longer.
I think it's ok to keep the old name to cover the simpler existing cases on which the auto-registering DP is fine: tests that will need this amount of control are gonna be fewer.

Reply:
Sure, will update.